frontend-react/src/App.js

```diff
@@ -5,10 +5,8 @@ import { createBrowserHistory } from 'history';
 import Container from 'react-bootstrap/Container';
 
 import routes, { renderRoutes } from './routes';
-import {
-  getMe,
-  login,
-} from './store/slices/auth';
+import { getMe, login } from './store/slices/auth';
+import LanguagePicker from "./components/LanguagePicker";
```

Comment:
I need to move `eslint` config to react project, however, to avoid too many changes in the future, please use single quotes for all strings